arbitrator/wasm-libraries/user-host/src/host.rs

```diff
@@ -39,14 +39,37 @@ pub unsafe extern "C" fn user_host__storage_store_bytes32(key: usize, value: usi
     let program = Program::start(2 * PTR_INK + EVM_API_INK);
     program.require_gas(evm::SSTORE_SENTRY_GAS).unwrap();
 
-    let api = &mut program.evm_api;
     let key = wavm::read_bytes32(key);
     let value = wavm::read_bytes32(value);
 
+    let api = &mut program.evm_api;
     let gas_cost = api.set_bytes32(key, value).unwrap();
     program.buy_gas(gas_cost).unwrap();
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn user_host__storage_transient_load_bytes32(key: usize, dest: usize) {
+    let program = Program::start(2 * PTR_INK + EVM_API_INK);
```

Comment:
Where does the magic number `2` come from?

Reply:
reading two ptr inputs (key, dest)